Assignment/solution_student/temp_converter/js/main.js

```diff
@@ -0,0 +1,10 @@
+$("#fahrenheit_to_celsius").on("click", function(){ 
+	var temp = $("#temperature").val();
+	$("#result").html((temp - 32) / 1.8);
```

Comment:
Very nice! Nice and lean! I love that you minimized the amount of work you needed to do and combined what could have been done in several steps into one. Great job! :+1: